### DISCUSSION TITLE
"Quick Shortcuts", "Open App On Boot", and "Autoclose" questions?

### DISCUSSION THREAD

**Discussion by phnord** [visnkmr/backgroundappslist#627] [Q&A]
I'm using the version currently available on the Amazon store.  Three quick questions:

1. In the Preference menu, there's a button that says "Enable Quick Shortcuts".  I cannot figure out what it does.  Can you help me?
2. "Open app on boot" – this doesn't seem to do anything when I boot?  I'm on an Insignia Fire TV if that helps
3. Your github profile makes mention of an "unreleased Autoclose version" – is it possible to obtain this?

**Comment by visnkmr** [comment]
> I'm using the version currently available on the Amazon store. Three quick questions:
> 
> 1. In the Preference menu, there's a button that says "Enable Quick Shortcuts".  I cannot figure out what it does.  Can you help me?

It shows a list of buttons that are shortcuts to open some of our other apps.
> 2. "Open app on boot" – this doesn't seem to do anything when I boot?  I'm on an Insignia Fire TV if that helps

This may not work on newer version of fire os since they do not allow apps to identfy device boot/ open a app without user opening the app, except for system apps embeded by amazon themselves.
> 3. Your github profile makes mention of an "unreleased Autoclose version" – is it possible to obtain this?

Sorry that one does not work on devices that don't implement accessibility API, fireos does not. Android TV do and it used to work in older version it was more of a proof of concept and required a bit of reverse engineering their UI for it to work.
